README.md

```diff
@@ -73,9 +120,11 @@ API for formatting code. Simple usage looks like this:
 
 ## Other resources
 
-* Before sending an email, see if you aren't asking a [frequently asked question][faq].
+* Before sending an email, see if you are asking a
+  [frequently asked question][faq].
 
-* Before filing a bug, of if you want to understand how work on the formatter is managed, see how we [track issues][].
+* Before filing a bug, of if you want to understand how work on the
+  formatter is managed, see how we [track issues][].
```

Comment:
I just added a couple leading spaces to improve readability in raw form. I don't have strong feelings, though.